docs/support/tutorials/python-usage-guide.md

```diff
@@ -0,0 +1,254 @@
+# Using Python on CSC supercomputers
+
+Some important aspects of working with the Python programming language
+are notably different on CSC supercomputers compared to usage on a personal
+device or in other HPC environments. To make the most of the computational
+resources available to you, it is helpful to be aware of the differences.
+
+See the
+[Python application page](../../apps/python.md)
+for general information on the Python language and
+pre-installed Python environments on CSC supercomputers.
+
+## Creating and managing Python environments
+
+### Installing Python packages to existing modules
+
+If there is a CSC-provided module that covers _almost_ everything you
+need, but it is missing a few Python packages, you can try installing
+those yourself with the pip package manager.
+
+See the [package lists on our Python application
+page](../../apps/python.md#pre-installed-python-environments) to find out
+which packages are installed in existing modules.
+If you think that some important package should be included by default
+in a module provided by CSC, don't hesitate to contact our [Service
+Desk](../contact.md).
+
+=== "Using `venv`"
+
+	The recommended way to add packages on top of an existing environment
+	is to use [venv](https://docs.python.org/3/tutorial/venv.html), which
+	is a standard Python module for creating a lightweight "virtual
+	environment". You can have multiple virtual environments, for example
+	one for each project.
+
+	For example to install a package called `whatshap` on top of the
+	CSC-provided [python-data](../../apps/python-data.md) module:
+
+	```bash
+	cd /projappl/<your_project>  # change this to the appropriate path for your project
```

Comment:
```suggestion
	cd /projappl/<your_project>/<venv_name>  # change this to the appropriate path for your project
```

Reply:
Actually I think the original is correct as the command `python3 -m venv --system-site-packages venv` will create the venv directory. Instead that command could be changed to:

`python3 -m venv --system-site-packages <venv_name>`

to make it clear that you can give it whatever name you want.

Reply:
Ok, sorry, if that is the case. But then it could be commented that venv creates this folder. (For example in Tykky empty folder must exist before.)

Reply:
Is the project `/projappl` directory the optimal place for creating venvs? At least VSCode automatically sources virtual environments found in an opened "folder", so instructing users to create them in whichever is their working directory might have its benefits

Reply:
/projappl is permanent during projects, from scratch in a half a year, it might get deleted. So in general it is recommended to use /projappl for installations.

Reply:
Makes sense :) I changed `venv` to `<venv_name>` and wrote in the info box below that there is no need to create a directory for the environment since `venv` takes care of it